platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -150,6 +150,20 @@ String OS_LinuxBSD::get_unique_id() const {
 	return machine_id;
 }
 
+int OS_LinuxBSD::get_physical_processor_count() const {
+	Ref<FileAccess> f = FileAccess::open("/proc/cpuinfo", FileAccess::READ);
```

Comment:
This probably won't work on BSD, so something `sysctl` based, similar to the macOS, probably should be added.

It seems to be `kern.smp.cores` for FreeBSD, not sure about other BSDs (maybe `hw.ncpu`).

Reply:
I've added code to support *BSD but I can't test it, since I currently don't have a VM setup active.